content/oci/oci.go

```diff
@@ -267,6 +322,13 @@ func (s *Store) loadIndexFile(ctx context.Context) error {
 //   - If AutoSaveIndex is set to false, it's the caller's responsibility
 //     to manually call this method when needed.
 func (s *Store) SaveIndex() error {
```

Comment:
Line 321 requires an update. This method is called on tag and manifest modifications.

Reply:
Then Line 45-51 also needs an update.

Reply:
Updated the doc comments in the file.